docs/gettinginvolved/infrastructure/release-schedule-versioning.rst

```diff
@@ -53,6 +53,15 @@ Minor marketing releases will be planned and scheduled by ClusterHQ's product te
 
 The version of a minor marketing release will have the micro version number incremented from the previous marketing release.
 
+Documentation Release
+^^^^^^^^^^^^^^^^^^^^^
+Documentation releases will be made when documentation for a major or minor marketing release is to be updated, without doing a full release.
+
+Documentation releases will be planned and scheduled by ClusterHQ's product team, in consultation with the marketing and engineering teams.
+
+The version of a documentation will have the version of the corresponding marketing release, with a ``+doc.X`` release, where ``X`` starts at ``1`` and is incremented for each pre-release.
```

Comment:
"incremented for each pre-release" > "incremented for each documentation release".